projects/organization-management/src/app/shared/user/user-profile-form/user-profile-form.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
```

Comment:
pls move this component to ...organization-management/src/app/components/user/user-profile-form and handle the tslint issue by writing a comment above the import (// tslint:disable-next-line...)

Later on we will change the appropriate lint rule to fix this.